test/types/fastify.test-d.ts

```diff
@@ -208,3 +210,5 @@ fastify().then(fastifyInstance => expectAssignable<FastifyInstance>(fastifyInsta
 expectAssignable<FastifyPluginAsync>(async () => {})
 expectAssignable<FastifyPluginCallback>(() => {})
 expectAssignable<FastifyPlugin>(() => {})
+
+expectAssignable<ValidationResult>({} as AjvErrorObject)
```

Comment:
Can you fulfill the object parameters?

Reply:
Thanks @RafaelGSS for reviewing

> Can you fulfill the object parameters?

By that, do you mean adding all properties of the ajv's `ErrorObject` interface to fastify's `ValidationResult` interface?

In that case, I would write something like:

```typescript
// fastify.d.ts
import { ErrorObject as AjvErrorObject } from 'ajv'

export interface ValidationResult extends AjvErrorObject {}
```

Let me know if that's what you're looking for. Thanks!

Reply:
I mean something like:
```ts
const ajvErrorObj = {
  keyword: '';
  instancePath: '';
  schemaPath: '';
  params: '';
  message?: '';
}
expectAssignable<ValidationResult>(ajvErrorObj)
```

Reply:
Thanks for the clarification.

I added the required properties to the ajv error object with [this commit](https://github.com/fastify/fastify/pull/4066/commits/4c63a8b420218c080b90536bef6661c4abceccc4).